readme.md

```diff
@@ -116,6 +151,11 @@ How is rebase different than a merge?
 
 Your Answer:
 ```text
+  In all seriousness, I had a dream about this last night, which is crazy.  The world depended on a merge to happen in the code and it kept not going through properly like what happened when we did the group project.  Finally I said with the most serious tone "we need to do a rebase" and the code merged to master and the world was saved
+
+  Merge takes all the commits from one branch and jam packs them into another branch usually the most current version of master
+
+  Rebase says forgot all those commits in the middle lets just jump from the now until master
```

Comment:
:+1:

Rebase doesn't really "forget" all those commits in the middle.  They aren't lost, history is just changed because rebase unwinds existing commits and adds them to the end.
BTW, thanks for saving the world.